### DISCUSSION TITLE
话说为什么我没启动steam++的时候可以访问githup，打开加速之后反而加载不出来了

### DISCUSSION THREAD

**Discussion by cyq20091029** [share121/inter-knot#4919] [其他]
哪位大佬能解答一下

**Comment by March-777th** [comment]
steam++是这样的，真的想流畅访问绳网，还是建议找个机场

**Comment by snivm** [comment]
自用，稳定，按流量计费，不限时长，流量用完前不会重置，每月不会刷新，支持​Netfli​​和chatgpt,最低​1元套​​，mojie.me/#/register?code=XdLgjGwe​，挂梯子打开

**Comment by manyshoes** [comment]
steam++最多访问一下，下载速度比裸连还慢，有需求还是用正经机场吧<img src="https://bbs-static.miyoushe.com/static/2023/11/16/2544d70667e060623d4b7702cd6bb3b9_8763505029750571668.png" alt="可琳-紧张" width="75" >

**Comment by lnx082** [comment]
最好使用联通
